setup.py

```diff
@@ -100,20 +114,12 @@
 
 dev_all = install_requires + dev_requires + docs_requires
 
-yatai_service = [
-    "grpcio~=1.34.0",  # match the grpcio-tools version used in yatai docker image
-    "google-cloud-storage",
-    "azure-cli",
-    "aws-sam-cli==0.33.1",
-    "psycopg2",
-    "psycopg2-binary",
-]
-
 extras_require = {
     "dev": dev_all,
-    "doc_builder": docs_requires + install_requires,  # required by readthedocs.io
     "test": test_requires,
-    "yatai_service": yatai_service + install_requires,
+    "yatai_service": yatai_service_requires,
+    "model_server": model_server_requires,
```

Comment:
FYI I'm adding "batch" for batch inference & "batch_distributed" for the distributed batch inference.

Reply:
@bojiang sounds great

Reply:
We may have "serve" or "inference_online" here?

Reply:
@bojiang only if we really need to separate those dependencies from `install_requires`. If a dependency is commonly used when developing or debugging a BentoService, we should just keep it in `install_requires`